form/src/main/java/org/dhis2/form/data/FormValueStore.kt

```diff
@@ -335,7 +343,11 @@ class FormValueStore(
 
     private fun saveFileResource(path: String): String {
         val file = FileResizerHelper.resizeFile(File(path), FileResizerHelper.Dimension.MEDIUM)
-        return d2.fileResourceModule().fileResources().blockingAdd(file)
+        return try {
```

Comment:
maybe you could add a breadcrumb to further investigations

Reply:
Yes I'll do that! Thanks